.github/dev-containers/devenv/README.md

```diff
@@ -0,0 +1,89 @@
+# Using this Docker Compose environment
+
+**NB!** The compose environment is meant only for testing X-Road in a development setup, it should never be used in a
+production environment.
+
+This document expects that you have a working Docker setup with Docker Compose installed on your system.
+
+The environment is set up by default to be used with the `xrddev-*` images available from the
+[main X-Road repository](https://github.com/orgs/nordic-institute/packages?repo_name=X-Road).
+
+The Compose environment also contains a Hurl container and scripts to initialize an environment with the following
+settings:
+
+* One Central Server
+* One management Security Server, which also acts as the producer Security Server for the example adapter under the
+  `DEV:COM:1234:TestService` subsystem. Permissions are given to the `DEV:COM:4321:TestClient` subsystem to access all
+  requests under it.
+* One consumer Security Server, which has the `DEV:COM:4321:TestClient` subsystem registered to it.
+* The `example-adapter` container. More information regarding it is available in
+  [its own repository](https://github.com/nordic-institute/xrd4j/tree/develop/example-adapter).
+
+The compose file has the following port mappings for the UI-s:
+
+* 4001 - Central Server
+* 4002 - Management Security Server
+* 4003 - Consumer Security Server
+* 8080 - Consumer Security Server client port
+* 8888 - TestCA
```

Comment:
Since not all the ports are for UIs, maybe it would be better to specify the target for each line separately, for example:

> The compose file has the following port mappings:
>
> * 4001 - Central Server UI
> * 4002 - Management Security Server UI
> * 4003 - Consumer Security Server UI
> * 8080 - Consumer Security Server client information system port
> * 8888 - TestCA UI

Reply:
Updated.